recipes/hole2/build.sh

```diff
@@ -0,0 +1,8 @@
+# compile HOLE with gfortran
+cd ${SRC_DIR}/src
+source ../source.apache
+
+FFLAGS+=" -fd-lines-as-comments "
+
+make FC=${GFORTRAN} CC=${CC_FOR_BUILD} CFLAGS="${CFLAGS}" FFLAGS="${FFLAGS}"
```

Comment:
CC_FOR_BUILD?

Reply:
Is this not the compiler we should be using? It's been a while, but it seems at point I believed that the [`GCC` variable was not set](https://github.com/ianmkenney/staged-recipes/commit/204f457f9370887572bbedb4b90c4c683aea0763)

Reply:
Use `CC` which is the compiler for the platform we are compiling for. `CC_FOR_BUILD` is the compiler for the platform we are building in.

Reply:
Switched it over in the next commit